lib/elixir/lib/inspect/algebra.ex

```diff
@@ -153,7 +161,10 @@ defmodule Inspect.Algebra do
 
   # Functional interface to "doc" records
 
-  @type t :: :doc_nil | :doc_line | doc_cons | doc_nest | doc_break | doc_group | binary
+  @type t :: :doc_nil | :doc_line | doc_cons | doc_nest | doc_break | doc_group
+             | doc_color | binary
```

Comment:
Please put the `|` in the previous line (I believe that's the codebase convention).